src/Workspaces/Core/Portable/FindSymbols/DeclaredSymbolInfo.cs

```diff
@@ -90,12 +96,12 @@ internal struct DeclaredSymbolInfo
             TextSpan span,
             ImmutableArray<string> inheritanceNames,
             int parameterCount = 0, int typeParameterCount = 0)
-            : this()
         {
-            Name = name;
-            NameSuffix = nameSuffix;
-            ContainerDisplayName = containerDisplayName;
-            FullyQualifiedContainerName = fullyQualifiedContainerName;
+            var stringTable = s_projectStringTable.GetOrCreateValue(project);
```

Comment:
➡️ A commit later in this PR, d1379c69de4d9627cd8e1b20ef8fe403c2efe0da, addresses my CWT concern.